docs/guide/cli.md

```diff
@@ -1,134 +1,76 @@
-| `--strictPort`            | Exit if specified port is already in use (`boolean`)                                                                                                                                  |
-| `--force`                 | Force the optimizer to ignore the cache and re-bundle (`boolean`)                                                                                                                     |
-| `-c, --config <file>`     | Use specified config file (`string`)                                                                                                                                                  |
-| `--base <path>`           | Public base path (default: `/`) (`string`)                                                                                                                                            |
-| `-l, --logLevel <level>`  | info \| warn \| error \| silent (`string`)                                                                                                                                            |
-| `--clearScreen`           | Allow/disable clear screen when logging (`boolean`)                                                                                                                                   |
-| `--configLoader <loader>` | Use `bundle` to bundle the config with Rolldown, or `runner` (experimental) to process it on the fly, or `native` (experimental) to load using the native runtime (default: `bundle`) |
-| `--profile`               | Start built-in Node.js inspector (check [Performance bottlenecks](/guide/troubleshooting#performance-bottlenecks))                                                                    |
-| `-d, --debug [feat]`      | Show debug logs (`string \| boolean`)                                                                                                                                                 |
-| `-f, --filter <filter>`   | Filter debug logs (`string`)                                                                                                                                                          |
-| `-m, --mode <mode>`       | Set env mode (`string`)                                                                                                                                                               |
-| `-h, --help`              | Display available CLI options                                                                                                                                                         |
-| `-v, --version`           | Display version number                                                                                                                                                                |
-
-## Build
-
-### `vite build`
+### Options
+
+<!-- render as html -->
+<div v-html="data.viteDevFlags"></div>
```

Comment:
One downside of using HTML tags is that the md files does not contain the content: https://deploy-preview-21270--vite-docs-main.netlify.app/guide/cli.md
I guess it's ok for now.